src/content/documentation/develop/differences-between-desktop-and-android-extensions.md

```diff
@@ -80,15 +80,15 @@ As a result of the UI differences, extensions for Firefox for Android do not sup
+The features of [`pageAction`][https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/pageAction/] are also reduced in Firefox for Android. The `manifest.json` key [page_action][https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/manifest.json/page_action] enables you to define the button icon and a popup. You then have use of [`pageAction.show()`](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/pageAction/show) and [`pageAction.hide()`][https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/pageAction/hide] however, once ‘shown’, note that the address bar button is visible in all tabs (unlike the desktop behavior, where the button is shown only for a specified tab.) But you will still be able to hide the pageAction using [`pageAction.hide()`][https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/pageAction/hide] on a specific tab (say, for example, you wish to hide your extension's page action icon in `about:addons` or `about:memory` tabs) And you can set a listener to [`pageAction.onClicked()`](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/pageAction/onClicked). [`pageAction.setPopup()`](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/pageAction/setPopup) and [`pageAction.getPopup()`](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/pageAction/getPopup) are also available, so you can update the popup or create a popup once the add-on is running.
 
-Also, in both [`pageAction`](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/pageAction) and [`browserAction`](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/browserAction) popup content is opened in a new tab and persists until the user manually closes the tab.
+Also, in both [`pageAction`][https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/pageAction/] and [`browserAction`][browser-action] popup content is opened in a new tab and persists until the user manually closes the tab.
```

Comment:
This adds a trailing slash which shouldn't be there. That link will 301 to the un-slashed url if it's included.